OpenRVS/classes/OpenServerList.uc

```diff
@@ -136,37 +133,31 @@ function ServerListSuccess(array<string> List)
 		//error checking for required fields
 		if ( (srv.ServerName == "") || (srv.IP == "") || (srv.GameMode == "") )
 			continue;//do not add this server
-		ServerList[ServerList.length] = srv;
+		servers[servers.length] = srv;
 	}
-	class'OpenLogger'.static.Info("loading" @ ServerList.Length @ "parsed servers", self);
 
-	M.ClearServerList();//clears the widget's server list
-	for (i=0; i<ServerList.Length; i++)
-	{
-		M.AddServerToList(ServerList[i].ServerName, ServerList[i].IP, ServerList[i].GameMode);
-	}
-	M.FinishedServers();//tells widget that the list is done, display them
+	return servers;
+}
 
-	ServerList.Remove(0, ServerList.Length);//all done, free the temp list
+// Retrieves fallback servers from local file on disk.
+function array<AServer> GetFallbackServers()
+{
+	local array<AServer> ServerList;//var name must match this in LoadConfig()

```

Comment:
I just had a thought, we removed Locked from the struct but if someone doesn't have an updated Servers.list will LoadConfig still load the rest of the struct from the ini file?

Reply:
Testing this shortly

Reply:
Works fine:
```
ScriptLog: [OpenRVS.OpenServerList] error: failed to retrieve servers over http
Warning: ImportText: Unknown struct item Locked in property FallbackServers
ScriptLog: [OpenRVS.OpenServerList] debug: loaded 48 servers from fallback file
ScriptLog: [OpenRVS.OpenServerList] info: loading 48 parsed servers
```
I added `Locked=false` to the 24/7 deathmatch server and this was the result.